CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Fixed
 
-- Nothing.
+- [#59](https://github.com/zendframework/zend-expressive-swoole/pull/59) Updated documentation adding reference to the missing `zend-expressive-swoole.swoole-http-server.options.pid_file` option.
```

Comment:
We usually don't do changelogs for documentation updates; documentation is built when merged to master. :smile:

Reply:
It actually makes sense, since doc changes won't affect anyone using the library. I have removed it :)